### ISSUE TITLE
Move gladereactor into twisted.manhole

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8156]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#441|
|Type|defect|
|Created|2003-12-18 23:50:55Z|

<details><summary>Searchable metadata</summary>

```
trac-id__441 441
type__defect defect
reporter__itamarst itamarst
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1071791455000000 1071791455000000
changetime__1470429412889694 1470429412889694
version__ 
owner__ 
cc__glyph cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
the reactor selection code in twisted.application.app doesn't require reactors
to be in twisted.internet, and manhole is a much more relevant place (and
prevents t.i depending on external packages).
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph commented|
|-|-|

```
#!html
<pre>
I've moved stuff in CVS.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 3a9498400eedeabdc5b938ac55461744f7d9c562
```
#!CommitTicketReference repository="" revision="3a9498400eedeabdc5b938ac55461744f7d9c562"
Merge pull request #441 from twisted/8706-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8706

Do not use inspect.getargspec() in twisted/test/test_twistd.py on Python 3
```
